qt/scientific_interfaces/Inelastic/QENSFitting/FunctionQParameters.h

```diff
@@ -12,6 +12,7 @@
 #include <string>
 #include <vector>
 
+typedef std::pair<std::string, std::size_t> pairNameSpectra;
 namespace MantidQt {
 namespace CustomInterfaces {
 namespace Inelastic {
```

Comment:
I think best to put this inside the `MantidQt::CustomInterfaces::Inelastic` namespace. Inside the annonymous namespace in the cpp file, we would then need `using namespace MantidQt::CustomInterfaces::Inelastic;`. It is ok to use `using namespace` in an enclosed scope

We also tend to use a capital letter for the first letter in a typedef such as here
https://github.com/mantidproject/mantid/blob/2ae50d1627844610269f2d8abae305fa68248811/Framework/MDAlgorithms/src/SmoothMD.cpp#L38